sources/models/src/lib.rs

```diff
@@ -127,6 +128,9 @@ struct KubernetesSettings {
     authentication_mode: KubernetesAuthenticationMode,
     bootstrap_token: KubernetesBootstrapToken,
     standalone_mode: bool,
+    eviction_hard: HashMap<EvictionHardKey, KubernetesThresholdValue>,
+    kube_reserved: HashMap<ReservedResourcesKey, KubernetesQuantityValue>,
```

Comment:
Let's prefix the types that are specific to Kubernetes.

```suggestion
    eviction_hard: HashMap<KubernetesEvictionHardKey, KubernetesThresholdValue>,
    kube_reserved: HashMap<KubernetesReservedResourceKey, KubernetesQuantityValue>,
```